File: 백준/Bronze/2292. 벌집/벌집.c
#include <stdio.h>

int main()
{
    int N, tmp = 1, answer = 1;
    scanf("%d", &N);
    
    while (tmp < N) {
        tmp += 6 * answer;
        answer += 1;
    }
    
    printf("%d", answer);
}

File: 백준/Bronze/2292. 벌집/벌집.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    
	    int N = sc.nextInt();
	    int tmp = 1, answer = 1;
	    
	    while (tmp < N) {
	        tmp += 6 * answer;
	        answer += 1;
	    }
	    
		System.out.println(answer);
	}
}